docs/release-info/release-notes.md

```diff
@@ -57,50 +57,741 @@ For tips about how to write a good release note, see [Release notes](https://git
+* Fixed issues with `ARRAY_CONTAINS` and `ARRAY_OVERLAP` with null left side arguments as well as `MV_CONTAINS` and `MV_OVERLAP` [#15974](https://github.com/apache/druid/pull/15974)
+* Fixed an issue which can occur when using schema auto-discovery on columns with a mix of array and scalar values and querying with scan queries [#16105](https://github.com/apache/druid/pull/16105)
+* Fixed windowed aggregates so that they update the aggregation value based on the final compute [#16244](https://github.com/apache/druid/pull/16244)
+* Fixed issues with the first/last vector aggregators [#16230](https://github.com/apache/druid/pull/16230)
+* Fixed an issue where groupBy queries that have `bit_xor() is null` return the wrong result [#16237](https://github.com/apache/druid/pull/16237)
+* Fixed an issue where Broker merge buffers get into a deadlock when multiple simultaneous queries use them [#15420](https://github.com/apache/druid/pull/15420)
+* Fixed a mapping issue in window functions where two nodes get the same reference [#16301](https://github.com/apache/druid/pull/16301)
+* Improved processing of index backed OR expressions [#16300](https://github.com/apache/druid/pull/16300)
+* Improved constant expression evaluation so that it's more thread-safe [#15694](https://github.com/apache/druid/pull/15694)
+* Improved performance for real-time queries using the MSQ task engine. Segments served by the same server are now grouped together, resulting in more efficient query handling [#15399](https://github.com/apache/druid/pull/15399)
+* Improved strict NON NULL return type checks [#16279](https://github.com/apache/druid/pull/16279)
+* Improved array handling for Booleans to account for queries such as `select array[true, false] from datasource` [#16093](https://github.com/apache/druid/pull/16093)
+* Improved how scalars work in arrays [#16311](https://github.com/apache/druid/pull/16311)
+* Improved LIKE filtering performance with multiple wildcards by not using `java.util.regex.Pattern` to match `%` [#16153](https://github.com/apache/druid/pull/16153)
+* Modified the `IndexedTable` to reject building the index on the complex types to prevent joining on complex types [#16349](https://github.com/apache/druid/pull/16349)
+* Restored `enableWindowing` context parameter for window functions [#16229](https://github.com/apache/druid/pull/16229)
+
 ### Cluster management
 
+#### Use cache for password hashes
+
+Improved Druid security by using a cache for password hashes while validating LDAP passwords.
+
+[#15993](https://github.com/apache/druid/pull/15993)
+
+#### Improved retrieving active task status
+
+Improved performance by reducing the number of metadata calls to retrieve the status of active tasks.
```

Comment:
```suggestion
Improved performance of the Overlord API `/indexer/v1/taskStatus` by serving status of active tasks from memory rather than querying the metadata.
```

Reply:
Updated.